back/app/services/side_fx_reaction_service.rb

```diff
@@ -23,6 +25,12 @@ def after_destroy(reaction, current_user)
 
   private
 
+  def lock_initiative_editing_if_required(reaction)
```

Comment:
Is it confusing to have two methods with same name in different side_fx files?

Reply:
On the contrary, it makes it clear that they do the same thing.